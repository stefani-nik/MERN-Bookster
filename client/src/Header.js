import React from 'react'
import logo from './logo.svg'

const Header = ({title}) => {
    return (
         <header className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <h1 className="App-title">{title}</h1>
          <a href="http://">Link</a>
        </header>
        );
}

export default Header;