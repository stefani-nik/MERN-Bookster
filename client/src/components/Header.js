import React from 'react';
import { Jumbotron, Container } from 'reactstrap';
import Login from './Login'

const Header = ({title}) => {
    return (
         <header className="App-header" >
         <Login/>
          <Jumbotron fluid>
             <Container fluid>
                <a href = "/">
                 <h1 className="app-title">{title}</h1>
                </a>
                 <h2 className="app-subtitle">Chatchy quote here</h2>        
               </Container>
            </Jumbotron>
        </header>
        );
}

export default Header;