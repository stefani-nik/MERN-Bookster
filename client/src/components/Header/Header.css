.app-header{
    height: 200px;
    background-image: url('../../images/header-bg.jpg');
    margin: 0 auto;
    padding-left: 50px;
    padding-right: 50px;
    padding-top: 30px;
    padding-bottom: 50px;
}