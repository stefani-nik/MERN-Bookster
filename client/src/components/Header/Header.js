import React from 'react';
import { Jumbotron, Container } from 'reactstrap';
import { Link } from 'react-router-dom';
import { Image } from 'semantic-ui-react'
import NavBar from '../NavBar/NavBar'
import SearchField from '../Search/Search'
import './header.css'

import logo from '../../images/logo.png'

const Header = (props) => {
    return (
         <header className="app-header" >
            <SearchField/>
            <Image className="header-logo" src={logo} alt="Bookster" size="medium" centered/>
            <NavBar/> 
        </header>
        );
}

export default Header;