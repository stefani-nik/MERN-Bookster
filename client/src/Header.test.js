import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe("header", ()=> {
    it("Should render correctly", ()=>{
        const wrapper = shallow(<Header title="Test title"/>);
        expect(wrapper).toMatchSnapshot();
    });
});