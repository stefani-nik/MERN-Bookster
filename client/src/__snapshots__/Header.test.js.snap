// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header Should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    title="Test title"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="logo"
          className="App-logo"
          src="logo.svg"
/>,
        <h1
          className="App-title"
>
          Test title
</h1>,
        <a
          href="http://"
>
          Link
</a>,
      ],
      "className": "App-header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "logo",
          "className": "App-logo",
          "src": "logo.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Test title",
          "className": "App-title",
        },
        "ref": null,
        "rendered": "Test title",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Link",
          "href": "http://",
        },
        "ref": null,
        "rendered": "Link",
        "type": "a",
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="logo"
            className="App-logo"
            src="logo.svg"
/>,
          <h1
            className="App-title"
>
            Test title
</h1>,
          <a
            href="http://"
>
            Link
</a>,
        ],
        "className": "App-header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "logo",
            "className": "App-logo",
            "src": "logo.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Test title",
            "className": "App-title",
          },
          "ref": null,
          "rendered": "Test title",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Link",
            "href": "http://",
          },
          "ref": null,
          "rendered": "Link",
          "type": "a",
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
